CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 3.4.3
+### Translations
```

Comment:
changes described bellow, are not related to "Translations" group in the main idea.